docs/decisions/0008-additional-redemption-metadata.rst

```diff
@@ -0,0 +1,55 @@
+0008 Additional Metadata for Policy Redemption
+************************************************************
+
+Status
+======
+
+Proposed
+
+Context
+=======
+
+This document intends to outline modifications to existing API endpoint(s) such that MFEs can pass additional metadata into the redemption flow. This extension was devised to support required GEAG meatadata such as DOB and terms acceptance dates.
+
+Decision
+=======
+
+The GEAG system requires additional metadata about a learner in order to process an allocation. This additional metadata (name, dob, etc) are collected during the enrollment flow on the edX side before enrollment. This additional metadata is not persisted anywhere on the edX side. Because of the lack of persistence and the placement of this information collection on the edX side of the process we decided to create a facility for the frontend to pass along additional metadata into the redemption flow.
```

Comment:
Does there need to be or should there be any validation on the `redeem` endpoint such that we can verify the necessary fields in `metadata` are passed in order to make the GEAG allocation before attempting to call `enterprise-subsidy`/GEAG? Or is the intent for this API to largely pass through the validation errors returned by GEAG?

Reply:
IMHO any validation involved with this data will be done by the subsidy service since has knowledge of the fulfillment systems - in this case enterprise access is mostly just passing it along and collecting the response to send back to the caller.

Reply:
Makes sense to me!